### ISSUE TITLE
network syncing issue with item dropping (Or at least guitars)

### ISSUE THREAD

**Issue by NilanthAnimosus** [Regalis11/Barotrauma#1226]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
I dropped a guitar on one side of the submarine, went to grab a random harpoon off a shelf and suddenly warped a guitar to my feet as a client host, curious I went to investigate further and found this infinitely repeatable, and thus a probable syncing error.

A second client of mine confirmed this.

**Steps To Reproduce**
Grab a guitar as a client host, drop the guitar. walk a long distance from it then pickup something else into your hands, this will drop the guitar you think your not holding.

**Version**
0.8.9.5 closed alpha

**Additional information**

https://i.gyazo.com/3f01458bdb3e93ecfe68f50ca43e8ba3.mp4

Right side is host client, left side is a second client:

![image](https://user-images.githubusercontent.com/29177976/53799600-a2937c00-3f32-11e9-87b2-0ac760ef6613.png)

After interacting with the weapon holder to grab a harpoon:
![image](https://user-images.githubusercontent.com/29177976/53799576-95768d00-3f32-11e9-89e6-9f9dbf07795a.png)

**Comment by DataCrusade** [comment]
Similar thing happened to me on the same version when I attempted to take a bandage out of a supply box  thingy.

**Comment by Regalis11** [comment]
Syncing the positions of the items is pretty broken in general in the current version, but I think the changes in 56bcb79 should take care of most of the issues (including this one). Still got to do some more testing before I'll merge the changes though.

**Comment by Regalis11** [comment]
I'm pretty sure this is the same problem as #1261. Will be fixed in the next patch!

**Comment by NilanthAnimosus** [comment]
I figured as much since it looks like its likely related to the dropping of equipped items, but im sure syncing has changed drastically since legacy

**Comment by NilanthAnimosus** [comment]
I am no longer seeing this particular desync (at least, if I drop an item, it most definately seems to drop).

However, I have died (and 2 others died with me) to what appeared to be item desync where a picked up item had not registered.

I picked up a diving mask (With oxygen tank) to a slot, and then equipped the mask onto my face by pressing the itemslots key it was in, this raised my oxygen bar and then it went down, it repeated this several times until going below zero as I attempted to tell my team on campaign  I was suffocating.

Upon finding my items placed into a cabinet the following round (probably from a fellow crewmate unless the game actually does this) there was a diving mask inside with a complete empty tank inside, I don't believe these were ever swapped out or used as it came out of a default emergency locker.

But, either the condition was incorrect, or someone placed a mask on me afterwards and I never took the mask, which it had two points to be incorrect (the pickup and the equip) without ever correcting the position of the item, if this was a diving suit or other item it would likely kill instantly, or weld without welding and such.

As a note - I WAS wearing a medical heatset. and swapped it to a mask, this has worked many times before, and I had picked the item up first.
